code/src/SessionManager.ts

```diff
@@ -17,14 +18,27 @@ export class SessionManager implements INotificationContext
 
     private lastSubId: number;
 
-    public constructor()
+    private notifyWithoutSubscriptions: boolean;
+
+    public constructor(notifyWithoutSubscriptions: boolean)
     {
+        this.notifyWithoutSubscriptions = notifyWithoutSubscriptions;
         this.lastSubId = 0;
         this.sessions = {};
     }
 
-    public Subscribe(oid: number) {
-        throw new Error('Method not implemented.');
+    public Subscribe(sessionId: number, oid: number)
```

Comment:
Ditto to the above comment on tsDoc:  Something like:

```suggestion
/**
   * Method that allows a caller to subscribe to notifications 
   *
   * @remarks
   * This constructor allows the caller receive all notifications regardless of subscriptions.
   *
   * @param sessionId  - The session id for the socket connection
   * @param oid  - The object id for the requested subscription
   * 
   */
    public Subscribe(sessionId: number, oid: number)
```